pkg/pillar/cmd/tpmmgr/tpmmgr.go

```diff
@@ -787,7 +797,7 @@ func testEcdhAES() error {
 	aesEncrypt(ciphertext, msg, encryptKey[:], iv)
 
 	recoveredMsg := make([]byte, len(ciphertext))
-	err = DecryptSecretWithEcdhKey(publicAX, publicAY, iv, ciphertext, recoveredMsg)
+	err = DecryptSecretWithEcdhKey(publicAX, publicAY, nil, iv, ciphertext, recoveredMsg)
```

Comment:
What does it mean to pass nil here? (I don't know how this test function is used).
A comment explaing why nil and what it does would be helpful

Reply:
added the comment.